README.md

```diff
@@ -4,4 +4,20 @@
 
 Utility classes to provide a better experience when running Appium tests in the TestObject cloud.
 
-For an introduction on how to use the utilities, please see our [documentation](https://help.testobject.com/docs/tools/appium/setups/suite-setup/junit/).
+
+## Installation
+
+TestObject Appium Java API Client is available from
+[Maven Central](https://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.testobject%22%20AND%20a%3A%22testobject-appium-java-api%22).
+
+    <dependency>
+      <groupId>org.testobject</groupId>
+      <artifactId>testobject-appium-java-api</artifactId>
+      <version>select a version</version>
+      <scope>test</scope>
+    </dependency>
+
+
+## Usage
+
+For an introduction on how to use the utilities, please see our [documentation](https://help.testobject.com/docs/tools/appium/setups/suite-setup/junit/).
```

Comment:
https://wiki.saucelabs.com/docs/tools/appium/setups/suite-setup/junit/

Reply:
Can we merge this pull request and then I create another one for fixing the link? The link is already wrong in the current Readme.

Reply:
I don't see why to create a new pr for it, but yes, you can.